MachObfuscator/Options/Options.swift

```diff
@@ -17,12 +17,12 @@ extension Options {
                          argv: CommandLine.arguments)
     }
 
-    init(argc: Int32, unsafeArgv: UnsafeArgv, argv: [String]) {
+    init(argc: Int32, unsafeArgv: UnsafeArgv, argv: [String], defaultManglerKey: String = "realWords") {
```

Comment:
What about moving `defaultManglerKey` to `static let defaultManglerKey = RealWordsMangler.key`? It is a little unsafe to use string literal when the const is available.